webapp/channels/src/components/post/post_component.tsx

```diff
@@ -513,7 +513,7 @@ const PostComponent = (props: Props): JSX.Element => {
             className={props.location === 'SEARCH' ? 'search-item__container' : undefined}
             data-testid={props.location === 'SEARCH' ? 'search-item-container' : undefined}
         >
-            {(isSearchResultItem || (props.location !== Locations.CENTER && props.isPinnedPosts) || props.isFlagged) && <DateSeparator date={currentPostDay}/>}
+            {(isSearchResultItem || (props.location !== Locations.CENTER && props.isPinnedPosts)) && <DateSeparator date={currentPostDay}/>}
```

Comment:
This might be close to working

```suggestion
            {(isSearchResultItem || (props.location !== Locations.CENTER && (props.isPinnedPosts || props.isFlaggedPosts))) && <DateSeparator date={currentPostDay}/>}
```

Reply:
I think `isFlaggedPosts` instead of the `isFlagged`

Reply:
Yep, that did it!